personifier.py

```diff
@@ -102,7 +102,10 @@ def rebuild_conversations(boc_df, dataframe):
                 Utterance(speaker, text)
                 for speaker, text in zip(replies["author_id"], replies["text"])
             ]
-            current_tweet = replies.iloc[-1]
+            try:
+                current_tweet = replies.iloc[-1]
+            except IndexError:
+                break
```

Comment:
Useful for the sample csv, where some tweets are not included and raises index error because it can't find the replies.